integration-sns-to-lambda/Function.cs

```diff
@@ -0,0 +1,53 @@
+using Amazon.Lambda.Core;
```

Comment:
How is this PR different to https://github.com/aws-samples/serverless-snippets/pull/54#pullrequestreview-1406253784 ?